src/components/EpisodesNav/renderUtils.jsx

```diff
@@ -37,10 +37,28 @@ function renderPaginationLink(data, index){
 
 	const episodeId = UTILS.convertToSafeString(title, "-");
 
-	const isActive = index === activeIndex;
+	const isActive 	= index === activeIndex;
+
+	let opacity = 1;
+	if(!isActive){
+		const decayDistance = 4;
+		const decayRate     = 1/decayDistance;
+		const fadeOffset    = Math.abs(index - activeIndex);
+		
+		opacity -= fadeOffset * decayRate;
+	}
+	
+	const offset = index - activeIndex;
+
+	const style = {
+		transform: `translateX(${offset}em)`,
+		opacity: opacity
+	};
 
 	return (
 		<li 
+			className={ isActive ? s.active : s.inactive }
```

Comment:
```suggestion
			className={`${s.item} ${isActive ? s.active : s.inactive}`}
```
I guarantee I'm going to see some funkiness below if there's not a shared className for both states, haha.  The reason we do this is that we want to create a 'rest' / 'shared' state and then define what is different FROM that state with the modifying states, if that makes sense?

Reply:
good shout!